docs/openAPI.json

```diff
@@ -994,7 +994,14 @@
                     "name": "return_format",
                     "description": "This query parameter formats the text that will be returned",
                     "schema": {
-                        "$ref": "#/components/schemas/return_format"
+                        "description": "This parameter formats the text that will be returned from the `v3` `texts/` API. It can have one of four values:\n1. `text_only` - This strips footnotes, inline references and all html tags from the returned text\n2. `strip_only_footnotes` - This only strips the footnotes and commentator <i> tags without stripping the other HTML tags.This is useful for the native app, where we do not display footnotes.\n3. `wrap_all_entities` - This wraps the HTML for links and topic links\n4. `default` - This returns the basic text as it’s saved in Sefaria’s DB\n",
+                        "type": "string",
```

Comment:
In order for this to render as expected (i.e. a dropdown for all enum values) we need to format this as follows:

```{
                    "examples": {
                        "Text Only": {
                            "value": "text_only"
                        },
                        "Wrap All Entities": {
                            "value": "wrap_all_entities"
                        }
                    },
                    "name": "return_format",
                    "description": "This parameter formats the text that will be returned from the `v3` `texts/` API. It can have one of four values:\n1. `text_only` - This strips footnotes, inline references and all html tags from the returned text\n2. `strip_only_footnotes` - This only strips the footnotes and commentator <i> tags without stripping the other HTML tags.This is useful for the native app, where we do not display footnotes.\n3. `wrap_all_entities` - This wraps the HTML for links and topic links\n4. `default` - This returns the basic text as it’s saved in Sefaria’s DB\n",
                    "schema": {
                        "enum": [
                            "default", 
                            "wrap_all_entities", 
                            "strip_only_footnotes", 
                            "text_only"
                        ],
                        "type": "string"
                    },
                    "in": "query"
                }
```


The main differences are:
1. The description outside the schema (noted in an early comment as well, apologies for the redundancy)
2. The `type` moved into the `enum` field but outside the array (i.e. to indicate that the overall type of this param is an `enum` of `string`s)

Reply:
Ok, 
1. Description was indeed in the wrong place by accident, but that means we have two description fields where one (coming from the defunct reusable ref) is much more detailed than the other. 
2. I'm not sure I understand, I followed https://swagger.io/docs/specification/data-models/enums/ and it clearly shows the type inside the schema as a sibling to the enum itself?

Reply:
1. Got it!
2. Hmmm, when I read this I see the `type` field nested inside the `enum` field, not adjacent. Maybe I'm misreading the yaml? Happy to go over together if that's better so we can get clarity. Thanks for following the spec so closely. 

Thanks!